roxie/ccd/ccdqueue.cpp

```diff
@@ -2845,10 +2845,11 @@ class RoxieUdpSocketQueueManager : public RoxieSocketQueueManager
         if (udpResendLostPackets && udpMaxSlotsPerClient > TRACKER_BITS)
             udpMaxSlotsPerClient = TRACKER_BITS;
         unsigned serverFlowPort = topology->getPropInt("@serverFlowPort", CCD_SERVER_FLOW_PORT);
+        bool sendFlowOnDataPort = topology->setPropBool("@sendDlowOnDataPort", true);
```

Comment:
sendDlowOnDataPort -> sendFlowOnDataPort, missing 'F'

Reply:
Also set that should be get... Both fixed